resources/js/common/components/PlayableHubsList/PlayableHubsList.tsx

```diff
@@ -44,3 +53,24 @@ export const PlayableHubsList: FC<PlayableHubsListProps> = ({ hubs }) => {
     </div>
   );
 };
+
+function filterHubs(
+  allHubs: App.Platform.Data.GameSet[],
+  options: Partial<{ canManageGames: boolean; excludeIds: number[] }>,
+): App.Platform.Data.GameSet[] {
+  const { canManageGames, excludeIds } = options;
+
+  return allHubs.filter((hub) => {
+    // Only users who can manage games can see team meta hubs on the list.
+    if (hub.title?.includes('Meta|') && !canManageGames) {
+      return false;
+    }
+
+    // There may be hub IDs marked for specific exclusion (such as those shown in game metadata).
+    if (excludeIds?.includes(hub.id)) {
+      return false;
+    }
+
+    return true;
+  });
+}
```

Comment:
When all hubs have been promoted, an empty section is shown.
http://localhost:64000/game2/6850
![image](https://github.com/user-attachments/assets/ae6a7a12-a7eb-4b04-839b-ea0461a5105d)

Reply:
Fixed in latest.